extensions/rocksdb-repos/FlowFileRepository.h

```diff
@@ -66,16 +66,14 @@ class FlowFileRepository : public ThreadedRepository, public SwapManager {
   }
 
   explicit FlowFileRepository(const std::string& repo_name = "",
-                     std::filesystem::path checkpoint_dir = FLOWFILE_CHECKPOINT_DIRECTORY,
+                     std::filesystem::path /*checkpoint_dir*/ = FLOWFILE_CHECKPOINT_DIRECTORY,
```

Comment:
could this parameter be removed?  as far as I can see, this constructor is only used in a few unit tests (and from the other constructor, but that wouldn't need to be changed)

Reply:
removed the parameter